src/components/Documents/AddButton.jsx

```diff
@@ -0,0 +1,56 @@
+/* global cozy */
+
+import React from 'react'
+import Button from 'cozy-ui/react/Button'
+import { Consumer } from 'components/ClientProvider'
+
+class _AddButton extends React.Component {
+  constructor() {
+    super()
+    this.handleChange = this.handleChange.bind(this)
+    this.create = this.create.bind(this)
+    this.state = { v2: false }
+  }
+  handleChange(ev) {
+    this.setState({ v2: ev.target.checked })
+  }
+
+  async create() {
+    const { client, doctype, attributes, onComplete } = this.props
+    const { v2 } = this.state
+    let response
+    if (v2) {
+      response = await client.mutate(client.create(doctype, attributes))
```

Comment:
`client.mutate()` [is expecting a `mutationDefinition`](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L374). We should directly call [`client.create`](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L181).

I think it's too expensive to maintain two clients here. Let's use CozyClient.